src/Monitor.cs

```diff
@@ -289,22 +289,25 @@ public async Task StopCaptureAsync(CancellationToken cancellationToken = default
             {
                 await Task.CompletedTask;
             }
-            else 
+            else
             {
-                await Task.Run(() => { _device.Close(); }, cancellationToken);
+                await Task.Run(() => { 
+                    _device.Close();
+                    _device.Dispose();
```

Comment:
Calling `Close()` and `Dispose()` actually causes the runtime to fail. Instead, just call `_device.StopCapture()`

Reply:
Per the documentation LibPcapLiveDevice is now disposable so that's why I called it.  I changed it to StopCapture only and will test.